product_docs/docs/efm/4/installing/prerequisites.mdx

```diff
@@ -17,6 +17,13 @@ Before configuring a Failover Manager cluster, you must satisfy the prerequisite
 
 Before using Failover Manager, you must first install Java (version 1.8 or later). Failover Manager is tested with OpenJDK, and we strongly recommend installing that version of Java. [Installation instructions for Java](https://openjdk.java.net/install/) are platform specific.
 
+!!! Note
+    There is a temporary issue with OpenJDK version 11 on RHEL and its derivatives. When starting Failover Manager, you may see an error like the following:
+
+    `java.lang.Error: java.io.FileNotFoundException: /usr/lib/jvm/java-11-openjdk-11.0.20.0.8-2.el8.x86_64/lib/tzdb.dat (No such file or directory)`
+
+    If so, the workaround is to manually install the missing package, e.g. `dnf install tzdata-java`
```

Comment:
```suggestion
    If so, the workaround is to manually install the missing package using the command `sudo dnf install tzdata-java`
```

Reply:
Centos 7 users would have to install dnf to run that exact command, but I'm ok with it. :)

Reply:
Could we add ("or `sudo yum install tzdata-java`")

Reply:
I don't think we need to. That's why I only had the one command as an example; if someone is still using centos 7 and doesn't know dnf=yum I'd be surprised. And I might be the only person still using centos 7.

Reply:
Ok, just wanted to make sure there wasn't underlying doubts.